### ISSUE TITLE
Dependency issues with --system-site-packages on Ubuntu 20.04

### ISSUE THREAD

**Issue by rkerr** [schorschii/LAPS4LINUX#38]
A slight weird one that I will confess to not entirely understanding. When installing with --system-site-packages on Ubuntu 20.04 there seem to be a number of issues due to dpapi-ng requiring dnspython >= 2.0.0 and cryptography >= 3.1 but the operating system supplied versions being older.

For dnspython pip will actually log this during the build but carry on regardless:

> ERROR: dpapi-ng 0.2.0 has requirement dnspython>=2.0.0, but you'll have dnspython 1.16.0 which is incompatible.

Which then results in an error at runtime:

> ModuleNotFoundError: No module named 'dns.asyncresolver'

For cryptography pip doesn't seem to notice, but there is an error at runtime:

> Encryption attempt failed __init__() missing 1 required positional argument: 'backend'

It seems that amending the laps-runner requirements.txt to specify the correct versions is enough to make pip actually install them.

**Comment by schorschii** [comment]
> It seems that amending the laps-runner requirements.txt to specify the correct versions is enough to make pip actually install them.

Yes, this will probably solve this issue. I don't test it on Ubuntu 20.04 anymore since support for this distro already ended, but if you make a pull request, I'll merge it.

**Comment by rkerr** [comment]
Done - think 20.04 is still in support for a few more months at least.

**Comment by schorschii** [comment]
You're right - Ubuntu 20.04 standard support ends in April 2025. Not much time left.
